californium-core/src/test/java/org/eclipse/californium/core/test/ObserveTest.java

```diff
@@ -100,8 +104,7 @@
 	private String uriX;
 	private String uriY;
 
-	private int notificationCounter = 0;
-	private int resetCounter = 0;
+	private AtomicInteger resetCounter = new AtomicInteger(0);
```

Comment:
I know, but isn't that the right thing to do, now that we are already dealing with this class' synchronization issues? It would only mean moving the property to `ServerMessageInterceptor`. Not such a big deal, is it?